### PR TITLE
OpenTracing support

### PR DESCRIPTION
**Pull request by dimtruck**
As of version 8.8.3.0 Repose supports [OpenTracing](http://opentracing.io).

This PR adds support for this service. For more information about Repose's
implementation of the spec, check out
http://www.openrepose.org/versions/latest/services/open-tracing.html

### CODE REVIEW

**Comment by dimtruck** [comment]
rfr @filler @mburns @itzg

File: attributes/default.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
   default['repose']['install_opts'] = '-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --force-yes'
 end
 
-default['repose']['version'] = '7.3.6.0'
+default['repose']['version'] = '8.8.3.0'
```
[the remark this answers is not in the document]
Comment (dimtruck) [reply attributes/default.rb]: By default, OpenTracing will be turned off.  I hope that's ok when you guys upgrade.

**Comment by filler** [comment]
Hello! 👋 

@dimtruck hit me up on Rack Slack about this PR.  I have not given this 👀 to date and likely will not for a few days.  If other maintainers like @mburns and @itzg have and find it is g2g, it is 👍 from me to 🚀!

**Comment by mburns** [comment]
Cut a release of head, which we haven't done in a long while.

These changes LGTM. Going to merge this to unblock downstream devs.

**Pull request merged**